src/AppBundle/Command/LoadCsvFixturesCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class LoadCsvFixturesCommand extends ContainerAwareCommand
```

Comment:
is it universal command for all fixtures? 
Must be LoadEmployeeCsvFixturesCommand, or something like that